### PR TITLE
Donations and misc

### PR DESCRIPTION
**Pull request by ubitux**

### CODE REVIEW

File: content/contribute.md [hunk 1]
```diff
@@ -7,8 +7,13 @@ weight: 20
...
 
 The simplest and most favored way to help the project is to **contribute
-financially**. We are currently working out the details; means to donate will
-soon be announced.
+financially** by making a donation through one of these services:
+
+- [Liberapay]({{< param liberapay >}})
+- [Ko-fi]({{< param kofi >}})
+
+Supporting us financially is essential to the project, so we're extremelly
```
Comment (mbouron) [inline content/contribute.md:15]: `extremely` ?

File: content/help.md [hunk 2]
```diff
@@ -10,5 +10,8 @@ weight: 30
 - [Bugs tracker]({{< param github >}}/nope.gl/issues) to report technical issues
 - [Contact us]({{< ref "/contact" >}})
 
+We also offer some **consulting services** for businesses. You can contact us by
```
[the remark this answers is not in the document]
Comment (mbouron) [reply content/help.md:13]: Maybe we can detail the key areas where we are proficient.

[the remark this answers is not in the document]
Comment (ubitux) [reply content/help.md]: Done

File: content/contribute.md (hunk 1 above)
Comment (ubitux) [reply content/contribute.md]: fixed

**Review by mbouron — APPROVED** [review]

**Pull request merged**